arelle/utils/validate/ValidationPlugin.py

```diff
@@ -99,6 +101,26 @@ def modelXbrlLoadComplete(
     ) -> ModelDocument | LoadingException | None:
         raise NotImplementedError
 
+    def validateFileSource(
+        self,
+        cntlr: Cntlr,
+        fileSource: FileSource,
+        *args: Any,
+        **kwargs: Any,
+    ) -> None:
```

Comment:
This hook doesn't get called for filings opened from the GUI, does it?

Also, this signature is missing the entry points passed from the CLI (could just not pass them since they're unused).

Reply:
Thanks for the explanation. I created #1848 as a followup.